query-node/package.json

```diff
@@ -23,14 +21,17 @@
     "db:processor:migrate": "hydra-processor migrate --env ../.env",
     "db:migrate": "yarn db:schema:migrate && yarn db:processor:migrate",
     "db:bootstrap": "yarn db:create && yarn db:prepare && yarn db:migrate",
+    "db:reset": "yarn db:drop && yarn db:prepare && yarn db:migrate",
     "bootstrap": "yarn codegen && yarn db:drop && yarn db:bootstrap",
-    "hydra-cli": "./codegen/node_modules/.bin/hydra-cli",
-    "hydra-typegen": "./codegen/node_modules/.bin/hydra-typegen",
+    "hydra-cli": "./codegen/node_modules/@joystream/hydra-cli/bin/run",
+    "hydra-typegen": "./codegen/node_modules/@joystream/hydra-typegen/bin/run",
     "codegen": "yarn hydra-cli codegen --schema ../../schemas/",
     "codegen:noinstall": "yarn codegen --no-install",
+    "typegen:configure": "NODE_URL=${NODE_URL:-ws://localhost:9000} envsub typegen.template.yml typegen.yml",
     "typegen": "rm -rf ./mappings/generated && yarn hydra-typegen typegen manifest.yml --debug",
     "mappings:build": "yarn workspace query-node-mappings build",
-    "format": "prettier ./ --write"
+    "start:dev": "./start.sh",
+    "kill:dev": "./kill.sh"
```

Comment:
These were part of the `giza` branch but were somehow missed in the merging process I believe.